getting_started/first_3d_game/01.game_setup.rst

```diff
@@ -29,6 +27,9 @@ Click *Import & Edit* to open the project in the editor.
 
 .. image:: img/01.game_setup/03.import_and_edit.png
 
+If you are prompted to convert the project, click *Convert Full Project*,
+and then *OK*. This will convert the project to the latest Godot version.
```

Comment:
The project files should be updated for Godot 4, so this shouldn't be required.

You may see a notice because you're upgrading the project's *minor* version, but the dialog you see is different in this case:

![image](https://github.com/godotengine/godot-docs/assets/180032/55295c21-4d9c-4440-bc15-afe7fc024cbc)

Reply:
Oh it gave me a warning about upgrading from a v3 project file. But I'm happy to revert this.